packages/react-core/src/components/Modal/ModalBoxBody.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/ModalBox/styles.css';
+
+const propTypes = {
+  /** content rendered inside the ModalBoxBody */
+  children: PropTypes.node,
+  /** additional classes added to the ModalBoxBody */
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  children: null,
+  className: ''
+};
+
+const ModalBoxBody = ({ children, className, ...props }) => (
+  <div
+    {...props}
+    className={css(styles.modalBoxBody, className)}
+    id="modal-description"
```

Comment:
Modal contents can vary, which means the element that has the id for the modal description could also vary, both in value and also where it's applied within the `.pf-c-modal-box__body` element.

The [pf-next examples](https://pf-next.com/components/ModalBox/examples/) show these two cases:
```
<div class="pf-c-modal-box__body" id="modal-description">
    To support screen reader user awareness of the dialog text, the dialog text is wrapped in a div that is referenced by aria-describedby.
</div>
```
and
```
<div class="pf-c-modal-box__body">
  <span id="modal-no-header-description">When static text describing the modal 
  is available, it can be wrapped with an ID referring to the modal's 
  aria-describedby value. Lorem ipsum ...</span>    Duis aute irure dolor ...
</div>
```

Reply:
@dmiller9911  I asked this somewhere else.  Do we generally have the consumer provide the id?